### PR TITLE
[UXE-4300] feat: add last updated to billing

### PR DESCRIPTION
**Pull request by lucasmendes21**
### What is the new behavior introduced by this PR?
add last updated to billing

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
image

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by peterpaulo-azion — APPROVED** [review]

File: src/tests/services/billing-services/load-invoice-last-updated-service.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const makeSut = () => {
+  const sut = loadInvoiceLastUpdatedService
+
+  return {
+    sut
+  }
+}
+
+describe('BillingServices', () => {
+  it('should return correct data on success', async () => {
```
Comment (pauloSF0) [inline src/tests/services/billing-services/load-invoice-last-updated-service.test.js:29]: Onde está o teste de GraphQl ?

File: src/services/billing-services/load-invoice-last-updated-service.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        groupBy: [createdDate]
+        filter: {
+          periodFromRange: {
+            begin: "${dateRange.dateInitial}", end: "${dateRange.dateFinal}"
+          }
+        }
+      ) {
+        last_updated: createdDate
+      }
+    }`
```
Comment (HerbertJulio) [inline src/services/billing-services/load-invoice-last-updated-service.js:21]: Não seria melhor remover o filtro, aplicar a ordenação por createdDate?
Além disso, recomendar adicionar limit: 1.

File: src/services/billing-services/load-invoice-last-updated-service.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+const adapt = (httpResponse) => {
+
+  const { body, statusCode } = httpResponse
+
+  return {
+    body: `Last Updated: ${formatDateToUSBilling(body?.data?.products[0].last_updated)}` ?? [],
+    statusCode
+  }
+}
```
Comment (HerbertJulio) [inline src/services/billing-services/load-invoice-last-updated-service.js:46]: Recomendo criar um retorno default ou algo do tipo;
Código de sugestão:  
```suggestion
const adapt = ({ body, statusCode }) => {

  const [bill] = body.data?.products || []
  const dataUpdated = formatDateToUSBilling(bill.last_updated) || '--'
  const formtText = dataUpdated ? `Last Updated: ${dataUpdated}` :  '---'
  return {
    body: formText,
    statusCode
  }
}
```

**Review by HerbertJulio — APPROVED** [review]

**Review by william-tome — APPROVED** [review]
LGTM

**Review by peterpaulo-azion — APPROVED** [review]

**Pull request merged**